recipes-core/cve-update/cve-update-nvd2-native.bb

```diff
@@ -208,7 +213,7 @@ def update_db_file(db_tmp_file, d, database_time):
                break
 
             # Recommended by NVD
```

Comment:
The comment "# Recommended by NVD" is at line 190, so it can be removed in this line.

Reply:
In referenced master, both have comments.

Reply:
Okay. I got it.